Add Beatmap Spotlights winter 2022 newspost & wiki article (#6855)
ppy/osu-wiki: wiki/Beatmap_Spotlights/Seasons/2022_Winter/en.md

---
tags:
  - charts
  - Ranking Charts
  - Seasonal Spotlights
---

> # Beatmap Spotlights Season 4 (Winter 2022)

Reviewer: ```suggestion
# Beatmap Spotlights Season 5 (Winter 2022)
```

## Schedule

| Event | Time |
| :-- | :-- |

> | Season Start | Feb 15th, 2022 |
> | Timeshift Lobby: Playlist A | Feb 15th - Feb 29th, 2022 |
> | Timeshift Lobby: Playlist B | Feb 29th - Mar 15th, 2022 |
> | Timeshift Lobby: Playlist C | Mar 15th - Mar 29th, 2022 |
> | Timeshift Lobby: Playlist A | Mar 29th - Apr 5th, 2022 |
> | Timeshift Lobby: Playlist B | Apr 5th - Apr 12th, 2022 |
> | Timeshift Lobby: Playlist C | Apr 12th - Apr 19th, 2022 |
> | Season Conclusion | Apr 19th, 2022 |

Reviewer: ```suggestion
| Season start | Feb 15, 2022 |
| Playlist lobby: Playlist A | Feb 15 - Feb 29, 2022 |
| Playlist lobby: Playlist B | Feb 29 - Mar 15, 2022 |
| Playlist lobby: Playlist C | Mar 15 - Mar 29, 2022 |
| Playlist lobby: Playlist A | Mar 29 - Apr 5, 2022 |
| Playlist lobby: Playlist B | Apr 5 - Apr 12, 2022 |
| Playlist lobby: Playlist C | Apr 12 - Apr 19, 2022 |
| Season conclusion | Apr 19, 2022 |
```

## Weekly multiplayer lobbies

As of now, joining the weekly multiplayer lobbies requires participants to download and install [osu!lazer](https://github.com/ppy/osu/releases).

1. Every 2 weeks of a season, a single playlist lobby is opened. Everyone can participate for the entire week.
   - After the first wave of lobbies, playlists will be opened again on a weekly basis.
2. Each playlist lobby provides a pre-defined playlist of beatmaps.
   - Each playlist contains an even distribution of Hard, Insane and Expert beatmaps.
   - Each playlist corresponds with the letter assigned to each 2 weeks. Therefore, each playlist gets replayed once during a season.
3. Within a playlist lobby, everyone competes towards a room-specific leaderboard.
   - Only the best score results on each beatmap within the playlist are summed up to create a weekly ranked score for each player.
   - Each beatmap of a playlist counts equally to the leaderboard score.
   - The individual beatmap score is based on osu!lazer's scoring system.
4. Once a playlist lobby concludes, the weekly ranked score of each participant of the room is added to the seasonal leaderboard.

## Playlists

> ### osu! (lead by [Nowaie](https://osu.ppy.sh/users/5428909))

Reviewer: ```suggestion
### osu! (led by [Nowaie](https://osu.ppy.sh/users/5428909))
```

- [Osamu Kubota - Presto \[Jval's Another\]](https://osu.ppy.sh/beatmapsets/1452332#osu/3005290), suggested by [Zelq](https://osu.ppy.sh/users/8953955)
- [Neru - Lost One no Goukoku \[eveless\]](https://osu.ppy.sh/beatmapsets/83547#osu/230641), suggested by [Ryuusei Aika](https://osu.ppy.sh/users/7777875)
- [OSTER project - Violet Rose \[Insane\]](https://osu.ppy.sh/beatmapsets/1178488#osu/2457698), suggested by [DigitalHypno](https://osu.ppy.sh/users/4384207)
- [onoken - ZADAMGA \[Luscent's Extra\]](https://osu.ppy.sh/beatmapsets/1492550#osu/3188917), suggested by [DigitalHypno](https://osu.ppy.sh/users/4384207) and [Lefafel](https://osu.ppy.sh/users/2295850)
- [UKRampage - Jack-the-Ripper \[Extravaganza\]](https://osu.ppy.sh/beatmapsets/1333932#osu/3274710), suggested by [Ryuusei Aika](https://osu.ppy.sh/users/7777875)
- [Nekomata Master - Byakuya Gentou \[EX\]](https://osu.ppy.sh/beatmapsets/36849#osu/119375), suggested by [Ryuusei Aika](https://osu.ppy.sh/users/7777875)
- [ELECTROCUTICA - Dependence Intension \[AQUARIUM+\]](https://osu.ppy.sh/beatmapsets/1402842#osu/2893781), suggested by [Riot](https://osu.ppy.sh/users/4256461)

> ### osu!taiko (lead by [OnosakiHito](https://osu.ppy.sh/users/290128))

Reviewer: ```suggestion
### osu!taiko (led by [OnosakiHito](https://osu.ppy.sh/users/290128))
```

- [Ikitama - sun and spirit \[oni\] (Alchyr)](https://osu.ppy.sh/beatmapsets/1413087#taiko/2954454), suggested by [Hivie](https://osu.ppy.sh/users/14102976)
- [Sven Noon - Difficult Mutations \[tasuke's Oni\] (tasuke912)](https://osu.ppy.sh/beatmapsets/935842#taiko/2583939), suggested by [uone](https://osu.ppy.sh/users/5321719)
- [Se-U-Ra - LOSHAXI -HACK- \[Oni\] (Genjuro)](https://osu.ppy.sh/beatmapsets/1094418#taiko/2287922), suggested by [uone](https://osu.ppy.sh/users/5321719)
- [The Flashbulb - That Missing Week (Alternate) \[Inner Oni\] (ZTH)](https://osu.ppy.sh/beatmapsets/931375#taiko/1944768), suggested by [uone](https://osu.ppy.sh/users/5321719)
- [ziki_7 - Leviathan \[Inner Oni\] (yassu-)](https://osu.ppy.sh/beatmapsets/1013884#taiko/2122025), suggested by [uone](https://osu.ppy.sh/users/5321719)
- [Digital Math - The Musky Thrust \[Inner Oni\] (Jaltzu)](https://osu.ppy.sh/beatmapsets/1153833#taiko/2408145), suggested by [uone](https://osu.ppy.sh/users/5321719)
- [aran - Mazy Metroplex \[Inner Oni\] (-Dusk)](https://osu.ppy.sh/beatmapsets/1449961#taiko/2981096), suggested by [uone](https://osu.ppy.sh/users/5321719)

> ### osu!catch (lead by [SadEgg](https://osu.ppy.sh/users/10278243))

Reviewer: ```suggestion
### osu!catch (led by [SadEgg](https://osu.ppy.sh/users/10278243))
```

- [glass beach - bedroom community \[girl who hates her life\] (Lacrima)](https://osu.ppy.sh/beatmapsets/1556085#fruits/3178862), suggested by [Rocma](https://osu.ppy.sh/users/566276)
- [tieff - Sunflower \[Rain\] (Hareimu)](https://osu.ppy.sh/beatmapsets/985106#fruits/2061108), suggested by [wonjae](https://osu.ppy.sh/users/5032045)
- [onumi - FAKE \[RAIN\] (JBHyperion)](https://osu.ppy.sh/beatmapsets/934785#fruits/2023545), suggested by [Rocma](https://osu.ppy.sh/users/566276)
- [dark cat - ELINE \[Catstep\] (Absolute Zero)](https://osu.ppy.sh/beatmapsets/1064394#fruits/2228562), suggested by [Rocma](https://osu.ppy.sh/users/566276)
- [Chroma - Made In Love \[Rei's MAXIMUM\] (Bunnrei)](https://osu.ppy.sh/beatmapsets/1028944#fruits/2151543), suggested by [wonjae](https://osu.ppy.sh/users/5032045)
- [Zekk - Feelsleft0ut \[rei's missing 0ut\] (Bunnrei)](https://osu.ppy.sh/beatmapsets/1145368#fruits/2391319), suggested by [SadEgg](https://osu.ppy.sh/users/10278243)
- [Phoebe Ryan - Mine (HANAEL Remix) \[Lost\] (Jemzuu)](https://osu.ppy.sh/beatmapsets/1222143#fruits/2542067), suggested by [Rocma](https://osu.ppy.sh/users/566276)

> ### osu!mania (led by [Hydria](https://osu.ppy.sh/users/808176), with help of [CrumpetFiddler](https://osu.ppy.sh/users/3518705) this season)

Reviewer: ```suggestion
### osu!mania (led by [Hydria](https://osu.ppy.sh/users/808176), with help from [CrumpetFiddler](https://osu.ppy.sh/users/3518705) this season)
```

> ### osu!mania (lead by [Hydria](https://osu.ppy.sh/users/808176), with help of [CrumpetFiddler](https://osu.ppy.sh/users/3518705) this season)

Reviewer: ```suggestion
### osu!mania (led by [Hydria](https://osu.ppy.sh/users/808176), with help of [CrumpetFiddler](https://osu.ppy.sh/users/3518705) this season)
```

---
tags:
  - charts
  - Ranking Charts
  - Seasonal Spotlights
---

> # Beatmap Spotlights Season 4 (Spring 2021)

Reviewer: wrong year and season

```suggestion
# Beatmap Spotlights Season 4 (Winter 2022)
```

## Schedule

| Event | Time |
| :-- | :-- |
| Season Start | Feb 15th, 2022 |
| Timeshift Lobby: Playlist A | Feb 15th - Feb 29th, 2022 |
| Timeshift Lobby: Playlist B | Feb 29th - Mar 15th, 2022 |
| Timeshift Lobby: Playlist C | Mar 15th - Mar 29th, 2022 |
| Timeshift Lobby: Playlist A | Mar 29th - Apr 5th, 2022 |
| Timeshift Lobby: Playlist B | Apr 5th - Apr 12th, 2022 |
| Timeshift Lobby: Playlist C | Apr 12th - Apr 19th, 2022 |
| Season Conclusion | Apr 19th, 2022 |

## Weekly multiplayer lobbies

As of now, joining the weekly multiplayer lobbies requires participants to download and install [osu!lazer](https://github.com/ppy/osu/releases).

> 1. Every 2 weeks of a season, a single timeshift multiplayer lobby is opened. Everyone can participate for the entire week.

Reviewer: ```suggestion
1. Every 2 weeks of a season, a single playlist lobby is opened. Everyone can participate for the entire week.
```

   - After the first wave of lobbies, playlists will be opened again on a weekly basis.

> 2. Each timeshift multiplayer lobby provides a pre-defined playlist of beatmaps.

Reviewer: ```suggestion
2. Each playlist lobby provides a pre-defined playlist of beatmaps.
```

>    - Each playlist is a subset of the whole roster of curated beatmaps for the current season.

Reviewer: a little too obvious to need pointing out

```suggestion
```

   - Each playlist contains an even distribution of Hard, Insane and Expert beatmaps.

>    - Each playlist corresponds with the letter assigned to each 2 weeks. Therefore, each playlist gets replayed once during a season.

Reviewer: ```suggestion
   - There are four playlists, A to D, which rotate once a week. Each playlist therefore gets re-played once during a season.
```

> 3. Within a timeshift lobby, everyone competes towards a room-specific leaderboard.

Reviewer: ```suggestion
3. Within a playlist lobby, everyone competes towards a room-specific leaderboard.
```

>    - Room leaderboards are based on the ranked score system. Therefore, only the best score results on each beatmap within the playlist are summed up to create a weekly ranked score for each user.

Reviewer: ```suggestion
   - Only the best score results on each beatmap within the playlist are summed up to create a weekly ranked score for each player.
```

>    - Each beatmap of a playlist counts equally to the leaderboard score. Harder difficulties do not provide more score.

Reviewer: tautology

```suggestion
   - Each beatmap of a playlist counts equally to the leaderboard score.
```

>    - Individual beatmap score is based on osu!lazer's scoring system.

Reviewer: ```suggestion
   - The individual beatmap score is based on osu!lazer's scoring system.
```

> 4. Once a timeshift lobby concludes, the weekly ranked score of each participant of the room is added to the seasonal leaderboard.

Reviewer: ```suggestion
4. Once a playlist lobby concludes, the weekly ranked score of each participant of the room is added to the seasonal leaderboard.
```

## Playlists

> 

Reviewer 2: ```suggestion

Each playlist in this season has their own theme. In the playlist A, spotlights team showcased beatmaps with unorthodox rhythm who are led by [Nowaie](https://osu.ppy.sh/users/5428909) (osu!), [OnosakiHito](https://osu.ppy.sh/users/290128) (osu!taiko), [SadEgg](https://osu.ppy.sh/users/10278243) (osu!catch), and [Hydria](https://osu.ppy.sh/users/808176) (osu!mania).

```

Reviewer: ```suggestion

The first playlist starting from this season has their own theme. In playlist A, the spotlights team showcases beatmaps with unorthodox rhythm, with picks led by [Nowaie](https://osu.ppy.sh/users/5428909) (osu!), [OnosakiHito](https://osu.ppy.sh/users/290128) (osu!taiko), [SadEgg](https://osu.ppy.sh/users/10278243) (osu!catch), and [Hydria](https://osu.ppy.sh/users/808176) (osu!mania).

```

> ### osu! (lead by [Nowaie](https://osu.ppy.sh/users/5428909))

Reviewer 2: ```suggestion
### osu!
```

> #### Playlist A (unorthodox rhythm)

Reviewer 2: ```suggestion
#### Playlist A
```

- [Osamu Kubota - Presto \[Jval's Another\]](https://osu.ppy.sh/beatmapsets/1452332#osu/3005290), suggested by [Zelq](https://osu.ppy.sh/users/8953955)
- [Neru - Lost One no Goukoku \[eveless\]](https://osu.ppy.sh/beatmapsets/83547#osu/230641), suggested by [Ryuusei Aika](https://osu.ppy.sh/users/7777875)
- [OSTER project - Violet Rose \[Insane\]](https://osu.ppy.sh/beatmapsets/1178488#osu/2457698), suggested by [DigitalHypno](https://osu.ppy.sh/users/4384207)
- [onoken - ZADAMGA \[Luscent's Extra\]](https://osu.ppy.sh/beatmapsets/1492550#osu/3188917), suggested by [DigitalHypno](https://osu.ppy.sh/users/4384207) and [Lefafel](https://osu.ppy.sh/users/2295850)
- [UKRampage - Jack-the-Ripper \[Extravaganza\]](https://osu.ppy.sh/beatmapsets/1333932#osu/3274710), suggested by [Ryuusei Aika](https://osu.ppy.sh/users/7777875)
- [Nekomata Master - Byakuya Gentou \[EX\]](https://osu.ppy.sh/beatmapsets/36849#osu/119375), suggested by [Ryuusei Aika](https://osu.ppy.sh/users/7777875)
- [ELECTROCUTICA - Dependence Intension \[AQUARIUM+\]](https://osu.ppy.sh/beatmapsets/1402842#osu/2893781), suggested by [Riot](https://osu.ppy.sh/users/4256461)

> ### osu!taiko (lead by [OnosakiHito](https://osu.ppy.sh/users/290128))

Reviewer 2: ```suggestion
### osu!taiko
```

> #### Playlist A (unorthodox rhythm)

Reviewer 2: ```suggestion
#### Playlist A
```

- [Ikitama - sun and spirit \[oni\] (Alchyr)](https://osu.ppy.sh/beatmapsets/1413087#taiko/2954454), suggested by [Hivie](https://osu.ppy.sh/users/14102976)
- [Sven Noon - Difficult Mutations \[tasuke's Oni\] (tasuke912)](https://osu.ppy.sh/beatmapsets/935842#taiko/2583939), suggested by [uone](https://osu.ppy.sh/users/5321719)
- [Se-U-Ra - LOSHAXI -HACK- \[Oni\] (Genjuro)](https://osu.ppy.sh/beatmapsets/1094418#taiko/2287922), suggested by [uone](https://osu.ppy.sh/users/5321719)
- [The Flashbulb - That Missing Week (Alternate) \[Inner Oni\] (ZTH)](https://osu.ppy.sh/beatmapsets/931375#taiko/1944768), suggested by [uone](https://osu.ppy.sh/users/5321719)
- [ziki_7 - Leviathan \[Inner Oni\] (yassu-)](https://osu.ppy.sh/beatmapsets/1013884#taiko/2122025), suggested by [uone](https://osu.ppy.sh/users/5321719)
- [Digital Math - The Musky Thrust \[Inner Oni\] (Jaltzu)](https://osu.ppy.sh/beatmapsets/1153833#taiko/2408145), suggested by [uone](https://osu.ppy.sh/users/5321719)
- [aran - Mazy Metroplex \[Inner Oni\] (-Dusk)](https://osu.ppy.sh/beatmapsets/1449961#taiko/2981096), suggested by [uone](https://osu.ppy.sh/users/5321719)

> ### osu!catch (lead by [SadEgg](https://osu.ppy.sh/users/10278243))

Reviewer 2: ```suggestion
### osu!catch
```

> #### Playlist A (unorthodox rhythm)

Reviewer 2: ```suggestion
#### Playlist A
```

- [glass beach - bedroom community \[girl who hates her life\] (Lacrima)](https://osu.ppy.sh/beatmapsets/1556085#fruits/3178862), suggested by [Rocma](https://osu.ppy.sh/users/566276)
- [tieff - Sunflower \[Rain\] (Hareimu)](https://osu.ppy.sh/beatmapsets/985106#fruits/2061108), suggested by [wonjae](https://osu.ppy.sh/users/5032045)
- [onumi - FAKE \[RAIN\] (JBHyperion)](https://osu.ppy.sh/beatmapsets/934785#fruits/2023545), suggested by [Rocma](https://osu.ppy.sh/users/566276)
- [dark cat - ELINE \[Catstep\] (Absolute Zero)](https://osu.ppy.sh/beatmapsets/1064394#fruits/2228562), suggested by [Rocma](https://osu.ppy.sh/users/566276)
- [Chroma - Made In Love \[Rei's MAXIMUM\] (Bunnrei)](https://osu.ppy.sh/beatmapsets/1028944#fruits/2151543), suggested by [wonjae](https://osu.ppy.sh/users/5032045)
- [Zekk - Feelsleft0ut \[rei's missing 0ut\] (Bunnrei)](https://osu.ppy.sh/beatmapsets/1145368#fruits/2391319), suggested by [SadEgg](https://osu.ppy.sh/users/10278243)
- [Phoebe Ryan - Mine (HANAEL Remix) \[Lost\] (Jemzuu)](https://osu.ppy.sh/beatmapsets/1222143#fruits/2542067), suggested by [Rocma](https://osu.ppy.sh/users/566276)

> ### osu!mania (lead by [Hydria](https://osu.ppy.sh/users/808176), with help of [CrumpetFiddler](https://osu.ppy.sh/users/3518705) this season)

Reviewer 2: ```suggestion
### osu!mania
```

> #### Playlist A (unorthodox rhythm)

Reviewer 2: ```suggestion
#### Playlist A
```

   - Each beatmap of a playlist counts equally to the leaderboard score. Harder difficulties do not provide more score.
   - Individual beatmap score is based on osu!lazer's scoring system.
4. Once a timeshift lobby concludes, the weekly ranked score of each participant of the room is added to the seasonal leaderboard.

## Playlists

### osu! (lead by [Nowaie](https://osu.ppy.sh/users/5428909))

#### Playlist A (unorthodox rhythm)

- [Polyphia - O.D. \[Hard\]](https://osu.ppy.sh/beatmapsets/1378026#osu/2850639), suggested by [Lefafel](https://osu.ppy.sh/users/2295850)
- [OSTER project - piano x forte \[Hard\]](https://osu.ppy.sh/beatmapsets/1296249#osu/2983635), suggested by [DigitalHypno](https://osu.ppy.sh/users/4384207)
- [Osamu Kubota - Presto \[Jval's Another\]](https://osu.ppy.sh/beatmapsets/1452332#osu/3005290), suggested by [Zelq](https://osu.ppy.sh/users/8953955)
- [Neru - Lost One no Goukoku \[eveless\]](https://osu.ppy.sh/beatmapsets/83547#osu/230641), suggested by [Ryuusei Aika](https://osu.ppy.sh/users/7777875)
- [OSTER project - Violet Rose \[Insane\]](https://osu.ppy.sh/beatmapsets/1178488#osu/2457698), suggested by [DigitalHypno](https://osu.ppy.sh/users/4384207)

> - [onoken - ZADAMGA \[Luscent's Extra\]](https://osu.ppy.sh/beatmapsets/1492550#osu/3188917), suggested by [DigitalHypno](https://osu.ppy.sh/users/4384207)

Reviewer 3: same as my last point
